Minor improvements

Feichtmeier opened the pull request:
- use orginal yaru green badges
![grafik](https://user-images.githubusercontent.com/15329494/87520042-b69cd100-c682-11ea-9f34-b1dbdd7b7e8d.png)

~~use a light terminal to reduce the heavy contrast against the bright window~~

Closes #28

Feichtmeier committed:
Minor improvements

- use orginal yaru green badges
- use a light terminal to reduce the heavy contrast against the bright window

Feichtmeier committed:
Further adaptions

- revert yaru light terminal colors back to white on purple
- exactly use the correct greens for light and dark yaru themes

Feichtmeier committed:
Use warning color for normal badges

rdnlsmith commented on themes/dark/yaru-dark.json:
```diff
@@ -356,9 +356,9 @@
     "list.inactiveSelectionBackground": "#575757",
     "activityBar.background": "#575757",
```
I'm thinking that `activityBar.background` and `input.background` should be darkened to `#484848` to improve contrast against the new badge color. I got `#484848` from the Software Center.

![badge-yaru-dark-contrast](https://user-images.githubusercontent.com/4039042/87612668-28076e80-c6d9-11ea-86b0-3e36276e5200.png)

rdnlsmith commented on themes/dark/yaru-dark.json:
```diff
@@ -356,9 +356,9 @@
+    "activityBarBadge.background": "#0c6d1a",
     "activityBarBadge.foreground": "#ffffff",
-    "badge.background": "#e95420",
+    "badge.background": "#991121",
```
This looks good, but I'm curious about the rationale? I don't think I've seen badges using this color while using Ubuntu.

Feichtmeier replied:
This change is more to the nature of that green in yaru. We use it as a "success" color, i.e. for new things that are not negative. That badges in the bottom panel are generally more of "attention!" things, mostly errors or warnings. So iff they are green it is the wrong message: success, you have an error :smile: 
Good Idea with the contrast, I'll change that for the dark theme!

rdnlsmith replied:
Ah, that definitely makes sense for the problems pane! They also show up on the search results pane, which is why I asked, but I think it's fine.

rdnlsmith reviewed (changes requested):
I like the new colors; I mentioned in another comment that I'd like to darken the activity bar of Yaru Dark for better contrast with the dark green badges.

I mentioned in the issue that the Yaru JSON files are generated from some TypeScript files. Would you mind moving your changes to `themes/src/light/yaru.ts` and `themes/src/dark/yaru-dark.ts`? Basically you would use the same key name but put it in the `overrides` section. One nice thing about this is you can also give names (_e.g._ `activityBarGray`, `badgeGreen`, _etc._) to the hex colors by adding them to the `colors` section.

To preview, (run `npm install` and also have `npx` installed, and then) run `npm run start`. Whenever you save one of the `.ts` files it will automatically re-generate, so you still get a live preview albeit with a bit of a delay. Let me know if you have any trouble with this&mdash;I don't believe anyone but me has attempted to run this since I set it up, and I only _kind of_ know what I'm doing when it comes to node.

Feichtmeier committed:
Yaru-dark: darken bar backgrounds

to improve contrast against the new badges

rdnlsmith committed:
Copy Feichtmeier's Yaru changes to .ts source files

rdnlsmith committed:
Improve legibility/consistency of Yaru selection

rdnlsmith committed:
Make Yaru Dark a little darker

Darken sidebar section headers so they continue to match the activity
bar, and darken the dropdown background so it continues to match the
input background. Darken the sidebar for better contrast with
new section header color and to match the Settings app sidebar in 20.04
Yaru Dark.

rdnlsmith commented:
@Feichtmeier Thanks!

I will likely publish an update in the next couple days. I usually tweet about it&mdash;would you like to be tagged in the tweet for attribution, and if so, what's your Twitter username?

Feichtmeier commented:
@rdnlsmith no twitter handle currently :) Thanks for the merge! :beer:
